### PR TITLE
run gitlabform for a single group and all its subgroups

### PR DESCRIPTION
**Pull request by exaV**
solves https://github.com/gitlabform/gitlabform/issues/613

This feature is disabled by default and can be enabled with the cli flag `--recurse-subgroups` or `-r`.

### CODE REVIEW

File: gitlabform/__init__.py
Comment (TimKnight-DWP) [inline gitlabform/__init__.py:58]: should default to False?

**Review by TimKnight-DWP — CHANGES_REQUESTED** [review]

File: gitlabform/__init__.py
Comment (exaV) [reply gitlabform/__init__.py:58]: depends on whether you want to make this the default (for cli) in the future. The tests currently pass with either True or False. If you want to go with True in the future (IMO you should) then i would leave it like that.

**Review by TimKnight-DWP — APPROVED** [review]

**Comment by exaV** [comment]
Are we waiting for another review? Please let me know if there's anything I can do me to get this merged

**Comment by TimKnight-DWP** [comment]
@exaV nope just waiting for me to remember 😓

**Pull request merged**
